.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+Thank you for your contribution.
+Before making a PR, please read our contributing guidelines at
+https://github.com/DevExpress/testcafe/blob/master/CONTRIBUTING.md#code-contribution
+-->
+
+## Purpose
+_Describe the problem you want to address or the feature you want to implement._
+
+## Approach
+_Describe how your changes address the issue or implement the desired functionality in as much detail as possible._
+
+## References
+_Provide a link to the existing issue(s), if any._
+
+## Pre-Merge TODO
+- [ ] Write tests for your proposed changes
+- [ ] Make sure that existing tests do not fail
+
+<!-- If your PR is not complete, mark it as [WIP] in its title -->
```

Comment:
And create your PR as a draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/

Reply:
Agree, added this info to the top of the template